lib/mayaUsd/fileio/writeJobContext.cpp

```diff
@@ -385,6 +385,39 @@ bool UsdMayaWriteJobContext::_NeedToTraverse(const MDagPath& curDag) const
         // If we're not going to create a prim at curDag, then we do not need to
         // traverse.
         return false;
+    } else {
```

Comment:
Given the if returns, I'd not put the extra check in an else to avoid nesting. Other previous checks are not nested inside elses.